src/components/InteractiveMap.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Map, TileLayer, GeoJSON } from 'react-leaflet';
+import '../containers/App.css';
+import '../data/geojson';
+import { geojson } from '../data/geojson';
+import MarkerClusterGroup from 'react-leaflet-markercluster';
+
+class InteractiveMap extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      center: [37.73, 14.2],
+      zoom: 8,
+      maxZoom: 18
+    };
+  }
+  render() {
+    return (
+      <div>
+        <Map center={this.state.center} zoom={this.state.zoom} maxZoom={this.state.maxZoom}>
+          <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
```

Comment:
We cannot use the Open Street Map tile server - see https://operations.osmfoundation.org/policies/tiles/. We have to chose another tile provider - Mapbox, Carto or Stamen?

Reply:
Using Stamen for now. Mapbox updated from Mapbox.js to Mapbox GL JS which is no longer built on leaflet. They deprecated mapbox-studio for developing with leaflet, now only works for the new Mapbox GL JS. Trying to get mapbox-studio-classic working to turn streets off and unneeded icons. Have to build the studio ourselves (https://github.com/mapbox/mapbox-studio-classic) but as of 10 days ago people have been running into issues building it. Sadly, it is no longer supported waiting to see if someone finds a fix in the community.